docs/source/development/debugging.md

```diff
@@ -4,12 +4,12 @@
 
 If you're running your Kedro pipeline from the CLI or you can't/don't want to run Kedro from within your IDE debugging framework, it can be hard to debug your Kedro pipeline or nodes. This is particularly frustrating because:
 
-* If you have long running nodes or pipelines, inserting `print` statements and running them multiple times quickly becomes a time-consuming procedure.
+* If you have long running nodes or pipelines, inserting `print` statements and running them multiple times quickly becomes time-consuming.
```

Comment:
This reminds me of a change I was going to make to this file also to do with rich logging! Will raise a PR for it.